### DISCUSSION TITLE
y 轴数据都为 0 时，如何将折线显示在底部 x 轴上而不是中间？

### DISCUSSION THREAD

**Discussion by sakura-flutter** [antvis/G2#6196] [Q&A] [answered]
### 问题描述

目前遇到数据都为 0 时，折现会显示在中间：
![image](https://github.com/antvis/G2/assets/28331128/fff76aa9-e141-44e1-9c7b-e1abd30d6c66)

尝试配置过 `domain` 限定 0 并加上 `clamp`，在数据都为 0 时可以达到预期行为：
![image](https://github.com/antvis/G2/assets/28331128/0c4b92da-69af-41c1-bc14-ba64f47fa196)

但如果有数据时会导致 y 轴定义域不是实际数据的值：
![image](https://github.com/antvis/G2/assets/28331128/a43c97a6-fce4-42f8-8f13-a48cc5b9f892)

### 重现链接

https://stackblitz.com/edit/react-tmslnz?file=src%2Findex.js

### 预期行为

数据都为 0 时折现显示在中间不太符合“直觉”，尤其是在多轴图中更为突出，而且一旦一个点有数据，整条折现变化很大。

预期是像 echarts 直接在 x 轴上显示：
![image](https://github.com/antvis/G2/assets/28331128/01e29aa9-9640-4d65-a9e5-562fcc64f721)
![image](https://github.com/antvis/G2/assets/28331128/93f9a908-454a-49a3-b6c9-8cf39aeb0302)

请问有没有在数据都为 0 时在底部显示且不影响 y 轴定义域范围显示的配置方式？
如果没有能否提供一个配置实现？感觉这个场景还是很常见的

感谢

### 补充说明（可选）

@antv/g2 Version: 5.1.15-beta.4

**Comment by pearmini** [comment, marked as answer]
所以这里比较好的做法去判断数据是否都为 0。如果都为 0，那么对 domain 等进行额外的配置，否则不配置。
